src/Pages/settings/messages.js

```diff
@@ -0,0 +1,60 @@
+import { defineMessages } from "react-intl";
+import { addIdAndDefaultMessageToMessages } from "Util";
+
+const messages = defineMessages({
+  Profile: "Profile",
+  Relays: "Relays",
+  Owner: "Owner",
+  Software: "Software",
+  Contact: "Contact",
+  Supports: "Supports",
+  Remove: "Remove",
+  Preferences: "Preferences",
+  Donate: "Donate",
+  LogOut: "Log Out",
+  Theme: "Theme",
+  System: "System (Default)",
+  Light: "Light",
+  Dark: "Dark",
+  AutoloadMedia: "Automatically load media",
+  AutoloadMediaHelp:
+    "Media in posts will automatically be shown for selected people, otherwise only the link will show",
+  None: "None",
+  FollowsOnly: "Follows only",
+  All: "All",
+  ImgProxy: "Image proxy service",
+  ImgProxyHelp: "Use imgproxy to compress images",
+  ServiceUrl: "Service URL",
+  ServiceKey: "Service Key",
+  ServiceSalt: "Service Salt",
+  EnableReactions: "Enable reactions",
+  EnableReactionsHelp:
+    "Reactions will be shown on every page, if disabled no reactions will be shown",
+  ConfirmReposts: "Confirm Reposts",
+  ConfirmRepostsHelp: "Reposts need to be manually confirmed",
+  ShowLatest: "Automatically show latest notes",
+  ShowLatestHelp: "Notes will stream in real time into global and posts tab",
+  FileUpload: "File upload service",
+  FileUploadHelp: "Pick which upload service you want to upload attachments to",
+  Default: "(Default)",
+  DebugMenus: "Debug Menus",
+  DebugMenusHelp: `Shows "Copy ID" and "Copy Event JSON" in the context menu on each message`,
+  EditProfile: "Edit Profile",
+  About: "About",
+  LnAddress: "LN Address",
+  Avatar: "Avatar",
+  Banner: "Banner",
+  Edit: "Edit",
+  PrivateKey: "Your Private Key Is (do not share this with anyone)",
+  Add: "Add",
+  AddRelays: "Add Relays",
+  Name: "Name",
+  Profile: "Profile",
+  Website: "Website",
+  Save: "Save",
+  DisplayName: "Display name",
+  Buy: "Buy",
+  Nip05: "NIP-05",
+});
+
+export default addIdAndDefaultMessageToMessages(messages, "Settings");
```

Comment:
`Settings` should be lowercase cause it needs to match the pathname exactly. In this example, the path should be `Pages.settings`.

```js
export default addIdAndDefaultMessageToMessages(messages, "settings");
```

This helper util shouldn't be necessary, but I couldn't figure out how to inject the `id` and `defaultMessage` using the existing babel tooling. IMO this helper is good enough. Only problem is you have to make sure the prefix matches exactly what the path is. This could be improved by programmatically generating the prefix instead of hardcoding it like this.

Reply:
I'll look into programmatically generating the prefix.